src/views/Card.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
+import { useParams } from 'react-router-dom/cjs/react-router-dom.min';
+import { getCardbyId } from '../services/cards';
+
+export default function Card() {
+  const [card, setCard] = useState([]);
```

Comment:
card will end up as an object, we should set the initial state value with that in mind